Makefile

```diff
@@ -0,0 +1,19 @@
+# makefile from the huggingface promptsource repository
+# https://github.com/bigscience-workshop/promptsource/blob/main/Makefile
+
+.PHONY: quality style
+
+check_dirs := promptsource
+
+# Check that source code meets quality standards
+
+quality:
+	black --check --line-length 119 --target-version py38 $(check_dirs)
```

Comment:
black recently announced a more stable versioning schedule https://black.readthedocs.io/en/stable/change_log.html  ... should we update the pinned version in dev requirements to `22.1.0` ?

Reply:
Yes, definitely! I have upgraded it. @hakunanatasha : Are you okay with this change?